nautobot/ipam/mixins.py

```diff
@@ -0,0 +1,27 @@
+class LocationFieldReplaceMixin:
```

Comment:
Maybe something like `LocationToLocationsQuerySetMixin` might be a more clear name?